Assets/00_Content/Scripts/Scenes/SceneInfo.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using UnityEngine;
+
+namespace Scenes {
+	[Serializable]
+	public class SceneInfo {
+		[SerializeField] public SceneReference scene;
+		[SerializeField] public AudioClip music;
+
+		public void Load() {
+			scene.Load();
+			// Tell music player to switch music / Play scene transition m.m
```

Comment:
So, if we have a way here for loading different scenes and playing their music, would it maybe be better if made the "between-rounds-menu" be a scene of it's own? Seeing as we will want to switch between them, and that it will have it's own music etc.
I had planned to have a "pause" function otherwise when I showed the score and canvas between rounds, but maybe it is better if we continue on this system that you've already started?

Reply:
I think having "between-rounds-menu" in the same scene makes more sense. Just put it on top of the game. This way we can do some cool animations in the background, like make the player characters dance or something when we win.